study-builder/studies/mpc/about-you.conf

```diff
@@ -231,15 +231,15 @@
         },
         {
           "question": {
-            include required("../snippets/date-question-month-year-dropdown.conf"),
```

Comment:
I initially created a date question when its supposed to be Y/N/DK PL question. corrected it.